lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -265,6 +265,7 @@ def create_instance(host, ami_spec, subnet_id)
       amitype = host['vmname'] || host['platform']
       amisize = host['amisize'] || 'm1.small'
       vpc_id = host['vpc_id'] || @options['vpc_id'] || nil
+      sg_cidr_ips = host['sg_cidr_ips'] ? host['sg_cidr_ips'].split(',') : ['0.0.0.0/0']
```

Comment:
Can this line be formatted like the others, i.e. by using `||` assignment instead of a ternary?

Reply:
I am not great with ruby. But I thought if I split on an nil value it would break